spec/controllers/registered_applications_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe RegisteredApplicationsController, type: :controller do
+
+end
```

Comment:
Tidy up by removing unused spec. These empty specs can be easily re-created/generated if/when needed.